packages/react-components/react-avatar/src/components/AvatarGroupItem/useAvatarGroupItemStyles.ts

```diff
@@ -1,42 +1,99 @@
+import { useSizeStyles } from '../../Avatar';
+import { AvatarGroupContext } from '../../contexts/AvatarGroupContext';
 import { makeStyles, mergeClasses, shorthands } from '@griffel/react';
-import { tokens } from '@fluentui/react-theme';
+import { tokens, typographyStyles } from '@fluentui/react-theme';
+import { useContextSelector } from '@fluentui/react-context-selector';
+import type { AvatarSizes } from '../../Avatar';
 import type { AvatarGroupProps } from '../../AvatarGroup';
 import type { AvatarGroupItemSlots, AvatarGroupItemState } from './AvatarGroupItem.types';
-import type { AvatarSizes } from '../../Avatar';
 import type { SlotClassNames } from '@fluentui/react-utilities';
-import { useContextSelector } from '@fluentui/react-context-selector';
-import { AvatarGroupContext } from '../../contexts/AvatarGroupContext';
 
 export const avatarGroupItemClassNames: SlotClassNames<AvatarGroupItemSlots> = {
   root: 'fui-AvatarGroupItem',
   avatar: 'fui-AvatarGroupItem__avatar',
   overflowLabel: 'fui-AvatarGroupItem__overflowLabel',
 };
 
+const avatarGroupItemDividerWidthVar = '--fuiAvatarGroupItem--divierWidth';
+
 /**
  * Styles for the root slot
  */
 const useRootStyles = makeStyles({
   base: {
     alignItems: 'center',
-    display: 'flex',
+    display: 'inline-flex',
+    flexShrink: 0,
     position: 'relative',
   },
   overflowItem: {
-    ...shorthands.padding(tokens.spacingVerticalXS, tokens.spacingVerticalXS),
+    ...shorthands.padding(tokens.spacingHorizontalXS),
   },
   nonOverflowItem: {
     ...shorthands.borderRadius(tokens.borderRadiusCircular),
   },
 });
 
+/**
+ * Styles for the avatar slot
+ */
+const useAvatarStyles = makeStyles({
+  nonOverflowItem: {
+    position: 'absolute',
+  },
```

Comment:
yes, for avatars that are smaller than size 24, they move out of the box unless I add the position absolute :/

Reply:
I'll do some more testing to figure it out, I think the problem is that Avatar is displayed as block because the root of AvatarGroupItem was in the right place, just the Avatar wasn't.